spec/blockings/write_spec.rb

```diff
@@ -15,15 +15,15 @@
 
       Fiber.schedule do
         File.open("./log1.txt", "w"){|f| f.write("aaa") }
-        puts '## finished writing in the first fiber'
+        # puts '## finished writing in the first fiber'
```

Comment:
Changes in this PR causes the runtime error when calling `puts` in `Fiber.schedule` blocks.
This would be fixed in another PR.